src/dex-ui-components/base/Dropdown/Dropdown.tsx

```diff
@@ -18,17 +19,19 @@ const DropdownSelector = (props: DropdownSelectProps) => {
   const { value, onChangeHandler, data } = props;
 
   return cloneElement(
-    <Select variant="dropDownSelector" value={value} icon={<ArrowDropDownIcon />} {...props.selectControls}>
-      {data
-        ? data.map((input: DropdownDataType) => {
-            return (
-              <option key={input.value} value={input.value}>
-                {input.label}
-              </option>
-            );
-          })
-        : undefined}
-    </Select>,
+    <Skeleton speed={0.4} fadeDuration={0} isLoaded={!props.isLoading}>
+      <Select variant="dropDownSelector" value={value} icon={<ArrowDropDownIcon />} {...props.selectControls}>
+        {data
```

Comment:
You can use the optional chaining for shorthand here: `data?.map`

Reply:
Done